00_Base/src/services/tariff.mapper.ts

```diff
@@ -63,4 +88,29 @@ export class TariffMapper {
       restrictions: undefined,
     };
   }
+
+  // TODO make flexible for more complicated tariffs
+  private mapTariffElementToCoreTariff(
+    tariffElements: TariffElement[],
+  ): Partial<Tariff> {
+    const tariffElement = tariffElements[0];
+    const priceComponents = tariffElement.price_components ?? [];
+    const pricePerKwh =
+      priceComponents.find((pc) => pc.type === TariffDimensionType.ENERGY)
+        ?.price ?? 0;
+    const pricePerMin =
+      (priceComponents.find((pc) => pc.type === TariffDimensionType.TIME)
+        ?.price ?? 0) / MINUTES_IN_HOUR;
+    const pricePerSession =
+      priceComponents.find((pc) => pc.type === TariffDimensionType.TIME)
```

Comment:
should this be `priceComponents.find((pc) => pc.type === TariffDimensionType.FLAT)` ?